gpii/node_modules/flowManager/src/Capture.js

```diff
@@ -0,0 +1,383 @@
+ *                           "magnification": 2
+ *                       }
+ *                   }
+ *               ]
+ *           },
+ *           {
+ *               "fakemag1": [
+ *                   {
+ *                       "settings": {
+ *                           "invert": true
+ *                       }
+ *                   }
+ *               ]
+ *           },
+ *           {
+ *               "fakemag2": [
+ *                   {
+ *                       "settings": {
+ *                           "magnification": 2,
+ *                           "invert": true
+ *                       }
+ *                   }
+ *               ]
+ *           }
+ *       ]
+ * '''
+ *
+ * and we want:
+ * '''json
+ *       {
+ *           "fakemag1": {
+ *               "magnification": 2,
+ *               "invert": true
+ *           },
+ *           "fakemag2": {
+ *               "magnification": 2,
+ *               "invert": true
+ *           }
+ *       }
+ * '''
+ *
+ * @param {Object} data - The raw captured data.
+ * @return {Object} Returns a new payload with collapsed data, and multiple settings handler
+ * results for the same solution merged together.
+ */
+gpii.flowManager.formatRawCapturedSettings = function (data) {
+    var togo = {};
+    fluid.each(data, function (sequenceItem) {
+        if (sequenceItem.isError) {
+            fluid.log("Error capturing settings for: ", sequenceItem);
+            return;
+        }
+        fluid.each(sequenceItem, function (item, key) {
+            if (!togo[key]) {
+                togo[key] = {};
+            }
+            fluid.each(item[0].settings, function (value, settingId) {
+                togo[key][settingId] = value;
+            });
+        });
+    });
+    return togo;
+};
+
+/**
+ * JAWS Voice Profile Adjustment GPII-4146
+ *
+ * When applying settings to a machine we create our own Voice Profile to use, which is
+ * hardcoded in the solutions registry as `GPII`. However, when capturing JAWS settings
+ * we want to use whatever the default voice profile is, if there is one. More so, if there
+ * is not a default voice profile, or a factory default is in use, we want to ignore the
+ * voice profile configuration.
+ *
+ * Note: While the JAWS solution entry is fairly stable, if there are changes to the settings
+ * configuration names, this will need to be updated.
+ *
+ * @param {Object} handlerSpec - The solutions registry entry for JAWS
+ * @param {Function} localResolver - The localResolver, usually from the `userSession` on the
+ * `lifecycleManager` to resolve the INI configuration paths where JAWS stores it's settings.
+ * @return {Object} Returns a new handlerSpec with the configurations adjusted depending on the
+ * current default voice profile on the system. This is a deep copy of `handlerSpec` the original
+ * is not modified.
+ */
+gpii.flowManager.jawsVoiceProfileAdjustment = function (handlerSpec, localResolver) {
```

Comment:
It's odd to have this specific function for Windows JAWS in the universal repo. Would "gpii/windows" repo be more appropriate for it? 

@amb26, any suggestions?

Reply:
With the exception of a small comment this has been removed from the universal code

Reply:
removed from universal